merge crvusd docs (#5)
CurveDocs/curve-docs: docs/LLAMMA/deployments.md

## **General Infrastructure**

> Due to testing-in-prod purposes, a few contract deployments took place, hence please ensure to utilize the correct and latest version. Latest deployment logs can be found [here](https://github.com/curvefi/curve-stablecoin/blob/master/deployment-logs/mainnet.log).

Reviewer: ```suggestion
Due to testing-in-prod purposes, a few contract deployments took place so please ensure that you are using the correct and latest version. Latest deployment logs can be found [here](https://github.com/curvefi/curve-stablecoin/blob/master/deployment-logs/mainnet.log).
```
